Sources/App/Core/RSS.swift

```diff
@@ -65,12 +65,16 @@ extension RSSFeed {
 }
 
 extension RecentPackage {
+    var rssGuid: String {
+        "\(repositoryOwner)/\(repositoryName)"
+    }
+
     var rssItem: Node<RSS.ChannelContext> {
         let link = SiteURL.package(.value(repositoryOwner),
                                    .value(repositoryName),
                                    .none).absoluteURL()
         return .item(
-            .guid(.text(link), .isPermaLink(true)),
+            .guid(.text(rssGuid), .isPermaLink(false)),
```

Comment:
It's interesting that the default value for an omitted `isPermaLink ` is true so we need to explicitly set false here.